README.md

```diff
@@ -127,11 +145,23 @@ Follow these steps to add Stormpath user authentication to your Express.js app.
   }
   ```
 
-  If the login attempt was successful, you will receive a 200 response and a session cookie will be set on the response. If an error occurred, we will send a 400 status with an error message in the body.
+  *Note: make sure that your client is setting the `Accept: application/json`
+  header on the request.*
+
+  *Using AngularJS?  Try our [Stormpath Angular SDK][]*
+
+  **API Services**
+
+  If your app is an API service that uses our client_crential workflow, your API consumers
+  can obtain access tokens by making this POST to your server:
 
-  **If your app is a website:**
+  ```
+  POST /oauth/token
+  Authorization: Basic <Base64Endoded(ACCOUNT_API_KEY_ID:ACCOUNT_API_KEY_SECRET);
+  Content-Type: application/x-www-form-urlencoded
 
-  Log them in by sending them to `/login` and fill in the login form.
+  grant_type=client_credentials
+  ```
 
   *[Read more about login in the documentation →](https://docs.stormpath.com/nodejs/express/latest/login.html)*
```

Comment:
We should move this link to the footer to be consistent, as you did with the Angular SDK link.